### ISSUE TITLE
Get TravisCI to work with gpiozero

### ISSUE THREAD

**Issue by georgemathieson** [georgemathieson/3D-Xmas-Tree-for-Raspberry-Pi#1]
At the moment, Travis will install gpiozero from pip but fails with: `ImportError: No module named gpiozero`.

Preferably this would be with Python3.

**Comment by AaronMulgrew** [comment]
Should be closed now as the build passes?

**Comment by georgemathieson** [comment]
Whilst the software does build, none of the stuff using gpiozero is tested. I'm yet to figure out how to get Travis to recognise the module (it does install via pip so I would have expected it to work, but it doesn't hence this issue).

**Comment by AaronMulgrew** [comment]
Perhaps try mocking the GPIO pins using something like this here?

http://gpiozero.readthedocs.io/en/stable/api_pins.html#mock-pins

**Comment by georgemathieson** [comment]
Right, but the mock factory is still part of gpiozero. The issue is getting Travis to recognise the import for gpiozero.

**Comment by AaronMulgrew** [comment]
https://stackoverflow.com/a/38630404 Have you tried something similar to this? Multiple issues reported between Python 2 and Python 3
